website/search/search.py

```diff
@@ -51,6 +51,10 @@ def update_user(user, index=None):
     index = index or settings.ELASTIC_INDEX
     search_engine.update_user(user, index=index)
 
+@requires_search
+def update_file(file_, index=None, delete=False):
+    index = index or settings.ELASTIC_INDEX
```

Comment:
Wouldn't `update_file(file, index=TEST_VALUE)` also allow you to control the value during tests?

Or is the issue that you want to set it globally once when tests start?